controllers/categories.js

```diff
@@ -1,11 +1,25 @@
-const db = require("../models/index");
-class CategoryController {
+const deleteOne = require("../services/category");
 
+const deleteOneCategory = async (req, res)=> {
 
+    try {
 
+        const {id} = req.params;
+        
+        await deleteOne(id);
+    
+        return res.json({
+            message: "deleted",
+            id
+        })
+        
+    } catch (error) {        
 
-
-  
+        return res.status(404).json({
+            error: true,
+            message: error.message
+        })
```

Comment:
como se sabe que el error es un `not-found 404` y no un `internal-server-error 500`? deleteOne deberia retornar una respuesta para saber si el usuario fue eliminado o no y en base a eso devolver un 404 o no.

Reply:
Okey, ya lo corregí